src/audio/ao_rtaudio.cpp

```diff
@@ -100,3 +132,9 @@ void AudioOutRt::errorCallback(RtAudioError::Type type, const std::string &error
     if (type != RtAudioError::WARNING && type != RtAudioError::DEBUG_WARNING)
         throw RtAudioError(errorText, type);
 }
+
+bool AudioOutRt::isCompatibleDevice(const RtAudio::DeviceInfo &info)
+{
+    // need at minimum 2 channels for output
+    return info.outputChannels >= 2;
```

Comment:
When 1 channel, why not to mix left and right then? For 2+, is need to put extra zeros, otherwise, a noisy crap would appear.

Reply:
Because in practice, there doesn't exist mono sound devices. (except of virtual ones, like a JACK client)
It's only to care for the case that somehow, a device in this case would be filled in the configuration. This may block the software from starting, not letting you to go to the configuration and change it.